README.md

```diff
@@ -158,6 +158,14 @@ performance.
 Hot reloading code is just one line in the beginning and one line in the end of
 each module so you might not need source maps at all.
 
+## React Native
+
+React Native
+**[supports hot reloading natively](https://facebook.github.io/react-native/blog/2016/03/24/introducing-hot-reloading.html)**
+as of version 0.22.  
+
+Using React Hot Loader with React Native can cause unexpected issues (see [#824](/../../issues/824)) and is not recommended.
```

Comment:
Just #824 should be fine, please remove link.

Reply:
Sorry to grave dig, but this doesn't actually link through to #824.
<img width="819" alt="Screen Shot 2019-05-05 at 9 31 46 AM" src="https://user-images.githubusercontent.com/12449668/57194766-b3537480-6f18-11e9-9fb0-7903082211b6.png">

Reply:
😢 just a year later.